src/controllers/features.js

```diff
@@ -0,0 +1,11 @@
+const {evaluate} = require('evaluate')
+const data = require('../../data.json')
+
+const get = (req, res) => {
+  const context = req.query
+  const features = evaluate(context, data.features)
```

Comment:
This evaluator looks like a middleware, what do you think?

Reply:
Good suggestion. Since I already did merge of this PR I'll be adding it in the following ones! Thanks.